spec/lib/dfid/patch/specialist_publisher/themes_spec.rb

```diff
@@ -38,5 +38,14 @@
         'value' => 'Neglected%20Tropical%20Diseases'
       )
     end
+
+    it 'sorts themes alphabetically by label' do
+      patch.run
```

Comment:
We're not `expect`ing `RDF::Repository` to receive `load` any more here (as in the test above on #L27). That means the test will be dependent on the internet for its triples.